website/src/components/SelfHostedSection.tsx

```diff
@@ -0,0 +1,68 @@
+import ArrowRightIcon from 'mdi-react/ArrowRightIcon'
+import FileDocumentMultipleOutlineIcon from 'mdi-react/FileDocumentMultipleOutlineIcon'
+import React, { FunctionComponent } from 'react'
+
+export const SelfHostedSection: FunctionComponent = () => {
+    const copyText = () => {
```

Comment:
Just a note for future us: it seems as though this would be worth extrapolating out since we use it a couple of times.

Reply:
I agree! I wanted to do this and I may just throw it in if I get a free moment but wanted to make sure I got the most important changes finished before tackling more tech debt. Also, execCommand is deprecated so I want to remove that and use something like [clipboard](https://www.npmjs.com/package/clipboard) instead.

Reply:
Oh yeah 100%

Reply:
@katjuell I ended up refactoring and consolidating this component 🤓